src/triage/cli.py

```diff
@@ -48,18 +50,20 @@ def __init__(self, parser):
         )
         parser.add_argument(
             '-s', '--setup',
-            help="file path to Python module to import before running the Experiment",
+            help=f"file path to Python module to import before running the "
+                 f"Experiment (default: {self.SETUP_FILE_DEFAULT})",
         )
 
     def setup(self):
-        setup_path = self.args.setup or os.path.abspath('experiment.py')
-        if setup_path is not None:
-            logging.info(f"Loading configurations from {setup_path}")
-            spec = importlib.util.spec_from_file_location("triage_config", setup_path)
-            triage_config = importlib.util.module_from_spec(spec)
-            spec.loader.exec_module(triage_config)
-            logging.info(f"Configuration loaded")
-        return None
+        if not self.args.setup and not os.path.exists(self.SETUP_FILE_DEFAULT):
+            return
+
+        setup_path = self.args.setup or self.SETUP_FILE_DEFAULT
```

Comment:
The logic I was recommending was:

* did the user specify a setup module path?
    * yes: use it. if it's wrong, fine, let the import fail, so they know what's going on.
    * no: ok, then use the default path, *if and only if it exists*.

The lines I've added express that, (though compactly).

What was merged attempts to consolidate this logic such that it wouldn't work this way, and rather it _always_ attempts to load the default path so long as a custom path hasn't been specified. (It does not check whether the default path exists.) This is not right to users.

Reply:
Makes a lot of sense!